src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -91,6 +91,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="DotNet.ReproducibleBuilds" Version="1.1.1">
```

Comment:
Let's only add [`Microsoft.SourceLink.GitHub`](https://www.nuget.org/packages/Microsoft.SourceLink.GitHub/). 

We don't need the other dependencies included  in `Dotnet.ReproducableBuilds`.

Reply:
@brminnick just adding the `SourceLink.GitHub` doesn't solve the issue of the Missing Symbols in Source Link, as you can see in the issue below
![image](https://user-images.githubusercontent.com/20712372/180890225-18a03545-9cc1-4d95-aa8f-a19f26af8642.png)

Reply:
Interesting. 

The way I've always done it is to call `dotnet pack`, which then generates two files a `.nupkg` file and a `.snupkg` file. 

I then upload both of these files to NuGet. 

Does this library somehow include rhe `.snupkg` inside of the `.nupkg`?

Reply:
@brminnick I think so... I followed the instructions [here](https://github.com/dotnet/sourcelink#using-source-link-in-net-projects) and it generates both packages... But they aren't linked... Will we need to do some extra steps on our CI? Tomorrow I can spec the nuget generated by our CI and see if it's all green

Reply:
I would expect it to generate a `.snupkg` file, and if it does, then we'll have to add ensure we copy it to the staging directory. 

I wonder if we can tweak the `filter` on this step to be `*nupkg`?

https://github.com/CommunityToolkit/Maui/blob/68b82412a7e31d961b7ec0da2909f38e2a07ff9a/azure-pipelines.yml#L161

Reply:
We copy both files. You can confirm it by downloading AzDO artifacts

Reply:
I remember from way long ago for Xamarin.Forms that NuGet didn't support embedded symbols yet so it had to be a separate package, I guess that has changed now? Not sure why we would want to choose one over the other though.

Reply:
@jfversluis when you release the package does it upload both nupkg and snupkg?

Reply:
Doesn't look like it...

Reply:
So, the package generated by this PR is all green, as you can see in the screenshot below:
![image](https://user-images.githubusercontent.com/20712372/181035360-f2bf5361-4874-49c4-97f5-1a1fbd623eed.png)

The good part is the package added on this PR doesn't leak to our users, so we should be safe.

Reply:
> Doesn't look like it...
> 
> ![image](https://user-images.githubusercontent.com/939291/180977559-6358118f-289d-4a84-9a16-8b27af324c6e.png)

probably if you replace the pattern to *nupkg it will upload both nupkg and snupkg

Reply:
I did that before and it would give the http conflict back as if I were trying to upload the same thing multiple times. Let’s see with the next release if it works in the wild, else Let’s figure out the next step